haystack/nodes/base.py

```diff
@@ -123,7 +123,9 @@ def get_params(self, return_defaults: bool = False) -> Dict[str, Any]:
     @classmethod
     def get_subclass(cls, component_type: str) -> Type[BaseComponent]:
         if component_type not in cls._subclasses.keys():
-            raise PipelineSchemaError(f"Haystack component with the name '{component_type}' not found.")
+            raise PipelineSchemaError(f"Haystack component with the name '{component_type}' not found. "
+                                      +"Install all OCR tools to use `PDFToTextConverter` using `pip install 'farm-haystack[ocr]'`" if component_type=='PDFToTextConverter' else "")
+
```

Comment:
Hey @in-balamurugan! Thank you for this contribution.

The `get_subclass` method is used to load all nodes, not only OCR related ones. Therefore, it would be nice to have a generic message here that can help users in different scenarios.

How about something like this?

```suggestion
            raise PipelineSchemaError(
                f"Haystack component with the name '{component_type}' not found. "
                "Check the class name of your component for spelling mistakes and make sure you installed "
                "Haystack with the proper extras: https://docs.haystack.deepset.ai/docs/installation#custom-installation"
            )
```